resources/js/dashboard/pages/administrator/forms/components/formField/FormFieldForm.vue

```diff
@@ -140,53 +149,86 @@ export default {
     },
     formStepsSorted () {
       return this.formSteps.slice().sort((a, b) => a.step - b.step)
+    },
+
+  },
+  watch: {
+
+    rowFormField () {
+      this.form = { ...this.rowFormField }
+      this.updating = true
+      this.btnTitle = 'Modifier'
+      this.title = 'Modifier un champ'
     }
+
   },
   methods: {
     ...mapActions([
       'formFieldTypeIndex',
-      'formFieldStore'
+      'formFieldStore',
+      'updateFormField'
     ]),
     onSubmit () {
       this.form.rules = this.fieldWillBeRequired ? 'required' : ''
       this.form.form_id = this.targetForm.id
 
       if (!this.form.form_field_order) {
         const MaxValue = this.targetForm.form_fields.flatMap(x => x.order_field)
-        console.log(MaxValue)
         this.form.order_field = MaxValue.length && MaxValue.length > 0 ? Math.max(...MaxValue) + 1 : 1
       }
-      this.formFieldStore(this.form)
-        .then(() => {
-          this.initForm()
-          this.$notify({
-            group: 'alert',
-            title: 'Champ rajouté avec succèss',
-            type: 'success'
+      if (this.btnTitle === 'Enregistrer') {
```

Comment:
Compare avec `updating` qui est un boolean, Ce titre : `enregistrer`, on peut le changer à tout bout champ par une autre personne et si il sait pas que tu l'as utilisé içi, ça va créer une régression

on devrait avoir  
`if (:this.updating) {`